pkg/runtime/runtime.go

```diff
@@ -253,6 +254,10 @@ type pubsubSubscribedMessage struct {
 // NewDaprRuntime returns a new runtime with the given runtime config and global config.
 func NewDaprRuntime(runtimeConfig *Config, globalConfig *config.Configuration, accessControlList *config.AccessControlList, resiliencyProvider resiliency.Provider) *DaprRuntime {
 	ctx, cancel := context.WithCancel(context.Background())
+	if globalConfig.Spec.WasmSpec.StrictSandbox != nil {
```

Comment:
Does this need to be added to the global context? How about adding it just to the context when we init a middleware component?

Reply:
Only add this config to context when init a middleware component and an output binding component(Currently only these two types have wasm components).